vnext/Desktop/packages.config

```diff
@@ -4,5 +4,5 @@
   <package id="ChakraCore.Debugger" version="0.0.0.42" targetFramework="native" />
   <package id="Microsoft.ChakraCore.vc140" version="1.11.13" targetFramework="native" developmentDependency="true" />
   <package id="Microsoft.Windows.CppWinRT" version="2.0.190730.2" targetFramework="native" />
-  <package id="ReactWindows.OpenSSL.StdCall.Static" version="1.0.2-p.2" targetFramework="native" />
+  <package id="ReactWindows.OpenSSL.v141.Static" version="1.1.1-d.1" targetFramework="native" />
```

Comment:
>d [](start = 64, length = 1)

(nit) would it be possible to just have more common version format, eg:
1.2.3.4

Reply:
Because of OpenSSL's particular versioning scheme, this would not work for us.
Next minor release will be `1.1.1e`, but we also need to be able to version our nugets, for fixes and incremental changes based on the exact same source base.

Plus, this convention is compliant with Semantic Versioning ("semver").

Becaue nothing about OpenSSL or its package is exported by RNW, this shouldn't have transcending impact.